multiversx/mx-api-service
# status checker initialize thresholds

## Reasoning
- Enhance adaptability across different environments (`mainnet`, `devnet`, `testnet`).
- Streamline configuration by embedding threshold settings within the service logic.
  
## Proposed Changes
- Adding a new method, `initializeThresholds`, in the `StatusCheckerService`, dynamically setting thresholds based on the network environment identified through apiConfigService. This method adjusts thresholds for:
- `Token, nodes, and providers counts.`
- `Token supply, assets, accounts, and transactions.`
- `Node validators count.`

## How to test
- Enable statusChecker flag and check for every network that all the values are set correctly
